src/components/Calculator.js

```diff
@@ -1,46 +1,58 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from "react";
```

Comment:
- [ ] Kindly don't disable linters, linters help us to keep standards and best practices, if errors arise let's try to fix them.